build.sbt

```diff
@@ -85,8 +85,6 @@ lazy val interfaces = crossProject(JVMPlatform, NativePlatform)
     moduleName := "scalafmt-interfaces",
     description :=
       "Dependency-free, pure Java public interfaces to integrate with Scalafmt through a build tool or editor plugin.",
-    crossVersion := CrossVersion.disabled,
-    autoScalaLibrary := false,
```

Comment:
what does this do? not needed for native, then?

Reply:
So it doesn't add Scala library for Java artifacts, since it will not be needed.